imx53-qsb: enable usb power

Enable usb power pin by using a fixed regulator. With this and the usb clock
patch is making usb working on my loco.

Signed-off-by: Arnaud Patard <arnaud.patard@rtp-net.org>
Index: linux/arch/arm/boot/dts/imx53-qsb.dts
===================================================================
--- linux.orig/arch/arm/boot/dts/imx53-qsb.dts	2013-05-20 00:46:34.000000000 +0200
+++ linux/arch/arm/boot/dts/imx53-qsb.dts	2013-05-20 01:06:31.000000000 +0200
@@ -66,6 +66,15 @@
 			regulator-max-microvolt = <3200000>;
 			regulator-always-on;
 		};
+
+		reg_usbpwr: usbpwr {
+			compatible = "regulator-fixed";
+			regulator-name = "usbpwr";
+			regulator-min-microvolt = <5000000>;
+			regulator-max-microvolt = <5000000>;
+			gpio = <&gpio7 8 0>;
+			regulator-always-on;
+		};
 	};
 
 	sound {
@@ -118,6 +127,7 @@
 				697  0x80000000	/* MX53_PAD_EIM_DA12__GPIO3_12 */
 				701  0x80000000	/* MX53_PAD_EIM_DA13__GPIO3_13 */
 				868  0x80000000	/* MX53_PAD_PATA_DA_0__GPIO7_6 */
+				878  0x80000000 /* MX53_PAD_PATA_DA_2__GPIO7_8 */
 				1149 0x80000000 /* MX53_PAD_GPIO_16__GPIO7_11 */
 			>;
 		};
